Article/Article.js

```diff
@@ -112,3 +124,54 @@ const data = [
   Step 5: Add a new article to the array. Make sure it is in the same format as the others. Refresh the page to see the new article.
 
 */
+
+const body = document.querySelector('body');
+
+
+
+
+
+function createArticles(title, date, firstParagraph, secondParagraph, thirdParagraph) {
+  const article = document.createElement('div');
+  const articleTitle = document.createElement('h2');
+  const articleDate = document.createElement('p');
+  const articleP1 = document.createElement('p');
+  const articleP2= document.createElement('p');
+  const articleP3 = document.createElement('p');
+  const button = document.createElement('span');
+
+
+
+  article.appendChild(articleTitle);
+  article.appendChild(articleDate);
+  article.appendChild(articleP1);
+  article.appendChild(articleP2);
+  article.appendChild(articleP3);
+  article.appendChild(button);
+
+
+  article.classList.add("article");
+  articleDate.classList.add("date");
+  button.classList.add("expandButton");
+
+
+  articleTitle.textContent = title;
+  articleDate.textContent = date;
+  articleP1.textContent = firstParagraph;
+  articleP2.textContent = secondParagraph;
+  articleP3.textContent = thirdParagraph;
+  button.textContent = "\u25BC";
+
+  button.addEventListener("click", (e) => {
+    console.log("clicked");
+    article.classList.toggle("article-open");
+    article.classList.toggle("close");
+  })
```

Comment:
I'm not sure you committed and pushed, but don't forget the `if` statement we tossed in here to change the button 💪